components/user_settings/general/user_settings_general.tsx

```diff
@@ -85,45 +83,93 @@ const holders = defineMessages({
     },
 });
 
-class UserSettingsGeneralTab extends React.PureComponent {
-    static propTypes = {
-        intl: intlShape.isRequired,
-        user: PropTypes.object.isRequired,
-        updateSection: PropTypes.func.isRequired,
-        updateTab: PropTypes.func.isRequired,
-        activeSection: PropTypes.string.isRequired,
-        closeModal: PropTypes.func.isRequired,
-        collapseModal: PropTypes.func.isRequired,
-        actions: PropTypes.shape({
-            logError: PropTypes.func.isRequired,
-            clearErrors: PropTypes.func.isRequired,
-            getMe: PropTypes.func.isRequired,
-            updateMe: PropTypes.func.isRequired,
-            sendVerificationEmail: PropTypes.func.isRequired,
-            setDefaultProfileImage: PropTypes.func.isRequired,
-            uploadProfileImage: PropTypes.func.isRequired,
-        }).isRequired,
-        requireEmailVerification: PropTypes.bool,
-        maxFileSize: PropTypes.number,
-        ldapFirstNameAttributeSet: PropTypes.bool,
-        ldapLastNameAttributeSet: PropTypes.bool,
-        samlFirstNameAttributeSet: PropTypes.bool,
-        samlLastNameAttributeSet: PropTypes.bool,
-        ldapNicknameAttributeSet: PropTypes.bool,
-        samlNicknameAttributeSet: PropTypes.bool,
-        ldapPositionAttributeSet: PropTypes.bool,
-        samlPositionAttributeSet: PropTypes.bool,
-        ldapPictureAttributeSet: PropTypes.bool,
-    }
+type Props = {
+    intl: IntlShape;
+    user: {
```

Comment:
We have a type in `mattermost-redux` called `UserProfile` that you can use here.

Reply:
@hiendinhngoc This is merged now, sorry about the delay :)

Reply:
@devinbinnie sorry, but still got error `Property 'password' does not exist on type 'UserProfile'` even the password was added to the library. What should I do to get the updated from `mattermost-redux`?

Reply:
You'll have to update the commit has in `package.json` and `package-lock.json` to the most recent commit on `master` from `mattermost-redux` and then run `npm i` on the webapp to get the updated package.